.vscode/settings.json

```diff
@@ -6,5 +6,8 @@
             ],
             "url": "./test/vwtest.schema.json"
         }
-    ]
+    ],
+    "files.associations": {
+        "cstdint": "cpp"
+    }
```

Comment:
Looks like an accidental add

Reply:
why is this directory even in the repo...?